zeebe/benchmarks/project/src/main/java/io/camunda/zeebe/Worker.java

```diff
@@ -66,6 +66,7 @@ public void run() {
             .jobType(jobType)
             .handler(handleJob(client, variables, completionDelay, requestFutures))
             .streamEnabled(isStreamEnabled)
+            .streamTimeout(Duration.ofMinutes(5))
```

Comment:
Just make sure to revert this 😅

Reply:
sure, just here for the benchmark, but I need a different branch for that anyway:
```
create Pod meg-31577-explicit-stream-timeout-handling-benchmark-zeebe-0 in StatefulSet meg-31577-explicit-stream-timeout-handling-benchmark-zeebe failed error: Po │
│ d "meg-31577-explicit-stream-timeout-handling-benchmark-zeebe-0" is invalid: metadata.labels: Invalid value: "meg-31577-explicit-stream-timeout-handling-benchmark-zeebe-7cd7b766d9": must be no more than 63 characters
```
😆